receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -106,7 +106,7 @@ func (ghs *githubScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 
 			branches, count, err := ghs.getBranches(ctx, genClient, name, trunk)
 			if err != nil {
-				ghs.logger.Sugar().Errorf("error %v getting branch count for repo %s", zap.Error(err), repo.Name)
+				ghs.logger.Sugar().Errorf("error getting branch count for repo %s", zap.Error(err), repo.Name)
```

Comment:
Does this log correctly? The template now accepts a single string argument instead of two. It looks like the repo name will be dropped or possibly appended to the end of the message.

Reply:
That's a good question. This was a suggestion from upstream. Looking at the functions in play, I think this should be reversed in order.